### PR TITLE
STO doc AIDA remediation advice DOC-3156

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- Feature

If tied to an Issue, list the Issue(s) here:

* [DOC-3156 STO doc :: AI-generated remediation recommendations](https://harness.atlassian.net/browse/DOC-3156)

## House Keeping
- [Final URL for topic](https://developer.harness.io/docs/security-testing-orchestration/use-sto/view-and-troubleshoot-vulnerabilities/ai-based-remediations)
- [Link to Slack thread with final instructions](https://harness.slack.com/archives/C05AYNC9Z7T/p1687266316947819)

### CODE REVIEW

File: docs/security-testing-orchestration/use-sto/view-and-troubleshoot-vulnerabilities/ai-based-remediations.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+</style -->
+
+```mdx-code-block
+import remediation_issue from '../static/ai-remediation-issue.png'
+import remediate_indiv_occurrences from '../static/ai-remediation-occurrences.png'
+import remediate_occurrence_three from '../static/ai-remediation-occurrence-3.png'
+import remediate_issue_refine from '../static/ai-remediation-issue-refine.png'
+```
+
+Harness AI Development Assistant (AIDA) auto-generates remediation advice for detected vulnerabilities. Harness AIDA explains the vulnerability, gives solutions how to fix it, and (where applicable) provides updated code snippets.
```
Comment (SudheendraKatte) [inline docs/security-testing-orchestration/use-sto/view-and-troubleshoot-vulnerabilities/ai-based-remediations.md:29]:
```suggestion
Harness AI Development Assistant (AIDA) auto-generates remediation advice for detected vulnerabilities. Harness AIDA explains the vulnerability, provides suggestions on how to fix it, and (where applicable) provides updated code snippets. 
```

File: docs/security-testing-orchestration/use-sto/view-and-troubleshoot-vulnerabilities/ai-based-remediations.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+   1. click **Edit Input**. 
+
+   2. Specify the occurrence, reference ID, and language (if you've scanned a codebase). 
+   
+     Harness AIDA can often auto-detect the language of a code snippet, but it's good practice to confirm that the language setting is correct. 
+
+   3. Add any additional context in the text pane. This might be a code snippet, a Dockerfile, or some other piece of information that's relevant to the specific target and issue. Then click **Generate**. 
+
+      :::note
```
Comment (SudheendraKatte) [inline docs/security-testing-orchestration/use-sto/view-and-troubleshoot-vulnerabilities/ai-based-remediations.md:68]:
```suggestion
      :::info note
```

**Review by SudheendraKatte — APPROVED** [review]

File: docs/security-testing-orchestration/use-sto/view-and-troubleshoot-vulnerabilities/ai-based-remediations.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+      ```mdx-code-block
+      <img src={remediation_issue} alt="Generate a new remediation using additional information." height="50%" width="50%" />
+      ```
+
+3. If you're not satisfied with the initial advice, click **Regenerate**.  
+
+2. If you want to regenerate the advice with additional information or context, do the following: 
+
+   1. click **Edit Input**.
```
Comment (KimberlyFields) [inline docs/security-testing-orchestration/use-sto/view-and-troubleshoot-vulnerabilities/ai-based-remediations.md:64]:
```suggestion
   1. Select **Edit Input**. 
```

**Review by KimberlyFields — COMMENTED** [review]
docs/security-testing-orchestration/use-sto/static/ai-copilot-issue-example.png contains "Copilot". Please remove that file. I also made some additional suggestions.

**Review by KimberlyFields — APPROVED** [review]

**Pull request merged**